symphony/app/.eslintrc.js

```diff
@@ -19,6 +19,9 @@ const combinedOpenSourcePattern = new RegExp(
 module.exports.extends = ['eslint-config-fbcnms'];
 module.exports.overrides = [
   {
+    env: {
+      node: true,
+    },
```

Comment:
This change will be applied to all files in Symphony, which is not preferable - it should be made at a more granular level.

Also, this is telling eslint that you're working in a node environment, whereas the changes in this PR only affect the client. Are you sure you need the node env?

Reply:
Yeah, there is 1 line of code in the workflow UI that accesses the 'process'. I think the line just displays the runtime version. So maybe instead of node this should have been 'browser'. 

Not sure, removed this for now.